workflow/scripts/read_filter.py

```diff
@@ -13,8 +13,10 @@
 def is_zipped(fname):
     return fname.endswith(".gz")
 
+if not isinstance(snakemake.config["min_length"], int):
+    raise TypeError(f"config.yml has no valid minimum read length, allowed are: {int}, but got {type(snakemake.config["min_length"])}.")
```

Comment:
sorry, ought to be WorkflowError, doesn't it? I mean, otherwise there will be traceback, hard to interpret, right?